recipe/meta.yaml

```diff
@@ -68,7 +68,6 @@ extra:
   recipe-maintainers:
     - mwcraig
     - bsipocz
-    - piti118
```

Comment:
@piti118 has never joined conda-forge, so let's go ahead and remove this so we stop getting the "added to conda-forge" emails...